build.bat

```diff
@@ -1,20 +1,35 @@
 @echo off
-cls
 
 .paket\paket.bootstrapper.exe
 if errorlevel 1 (
+  echo "Bootstrapper failed"
   exit /b %errorlevel%
 )
 
 .paket\paket.exe restore
 if errorlevel 1 (
+  echo "Paket restore failed"
   exit /b %errorlevel%
 )
 
-SET NuGetTool=packages\NuGet.CommandLine\tools\NuGet.exe
-SET EnableNuGetPackageRestore=true
+SET NuGetTool=%~dp0packages\NuGet.CommandLine\tools\NuGet.exe
 
-msbuild TickSpec.sln /p:Configuration=Release
-cd Nuget\dotNet
-..\..\%NuGetTool% pack TickSpec.nuspec
-cd ..\..
\ No newline at end of file
+msbuild TickSpec.sln /p:Configuration=Release /m
+
+if errorlevel 1 (
+  echo "Build failed"
+  exit /b %errorlevel%
+)
+
+if %1.==. (
+    set version="1.0.1.1"
+) else (
+    set version="%1"
+)
+
+.paket\paket.exe pack packed_nugets --version %version% --symbols
+
+pushd %~dp0packed_nugets
+%NuGetTool% pack ..\Nuget\NUnit\TickSpec.NUnit.nuspec -Version %version%
+%NuGetTool% pack ..\Nuget\xUnit\TickSpec.xUnit.nuspec -Version %version%
```

Comment:
Maybe we should drop this - few people use xUnit v1 and it's not helpful to send people the wrong way (we might need to unlist the package ... or maybe we should log an issue to do a package that has all the helpers ready to go for xUnit v2 without the sample - obviously this was all done before Paket so for example there are source files in the package which are intended to be edited, which is not a good pattern)

Reply:
I commented it out. We should probably consider how to provide such thing (does it really need a nuget)? For example, the NUnit variant will not work even for NUnit3. I agree that handling it as a separate issue is good way to go.

Reply:
Yeah - if you look on the DI branch, I have a cleaner way to to the NUnit linking (based onthe xUnit style) which does not bind to the specific version of NUnit (i.e. no base type).
Will focus on getting DI merged first but happy to discuss the approach on an Issue if you want